index.js

```diff
@@ -20,37 +23,35 @@ module.exports = {
         'No Applitools API key found! Set APPLITOOLS_API_KEY with your API key from https://eyes.applitools.com',
       );
     }
-    
-    const applitoolsConfig = {
-      showLogs: true
-    }
-    
-    fs.writeFile(path.resolve(PUBLISH_DIR, '..', 'applitools.config.js'), `module.exports = ${JSON.stringify(applitoolsConfig)}`, 'utf8');
 
     const port = 9919;
     const server = http
       .createServer(ecstatic({ root: `${PUBLISH_DIR}` }))
       .listen(port);
 
-    fs.copy(
-      path.resolve(__dirname, 'template/visual-diff.json'),
-      path.resolve(PUBLISH_DIR, '..', 'visual-diff.json'),
-    );
+    const applitoolsConfig = {
+      showLogs: true,
+    };
 
-    fs.copy(
-      path.resolve(__dirname, 'template/visual-diff'),
-      path.resolve(PUBLISH_DIR, '..', 'visual-diff'),
-    );
+    await Promise.all([
+      fs.writeFile(
+        APPLITOOLS_CONFIG,
+        `module.exports = ${JSON.stringify(applitoolsConfig)}`,
+        'utf8',
+      ),
+      fs.copy(`${__dirname}/template/${VISUAL_DIFF}`, VISUAL_DIFF),
+    ]);
 
     const cypress = require('cypress');
     const builtPages = glob
       .sync(`${PUBLISH_DIR}/**/*.html`)
       .map((p) => path.dirname(p.replace(PUBLISH_DIR, '')));
 
     const results = await cypress.run({
-      configFile: 'visual-diff.json',
-      config: { baseUrl: `http://localhost:${port}` },
+      configFile: `${VISUAL_DIFF}/visual-diff.json`,
+      config: { baseUrl: `http://localhost:${port}`, video: false },
       env: {
+        SITE_NAME: process.env.SITE_NAME || 'localhost-test',
```

Comment:
This was missing and as a result in the `Applitools` UI it always showed `localhost-test`